src/bundle/Resources/views/dashboard/tab/all_media.html.twig

```diff
@@ -17,8 +17,12 @@
                 <td><a href="{{ url('_ez_content_view', { 'contentId': row.contentId }) }}">{{ row.name }}</a></td>
                 <td>{{ row.type }}</td>
                 <td>
-                    {{ ez_render_field(row.contributor, 'first_name') }}
-                    {{ ez_render_field(row.contributor, 'last_name') }}
+                    {% if row.contributor is instanceof('eZ\\Publish\\API\\Repository\\Values\\User\\User') %}
```

Comment:
We don't need a plugin. `{% if row.contributor is defined %}` would be sufficient here.